django/transformer/transform/transformer.py

```diff
@@ -17,7 +18,8 @@ def compute_fhir_object_id(mapping_id, primary_key_value) -> str:
     :return: UUIDv5
     """
     logical_reference = UUID(mapping_id, version=4)
-    return str(uuid5(logical_reference, str(primary_key_value)))
+    normalized_primary_key_value = normalize_to_str(primary_key_value)
```

Comment:
Why not raising here if `normalized_primary_key_value` isn't properly formatted?